### DISCUSSION TITLE
ko/guides/kfold-cross-validation/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#15797] [Docs]
# ko/guides/kfold-cross-validation/

Ultralytics YOLO 을 사용하여 객체 감지 데이터 세트에 대한 K-Fold 교차 검증을 구현하는 방법을 알아보세요. 모델의 신뢰성과 견고성을 향상하세요.

https://docs.ultralytics.com/ko/guides/kfold-cross-validation/

**Comment by sangeun-jo** [comment]
K-폴드 데이터 세트 분할  4번에서 `images.extend(sorted((dataset_path / "images").rglob(f"*{ext}")))` 이 코드를  `images.extend(sorted(dataset_path.rglob(f"*images/*{ext}")))` 로 변환해야 작동했습니다.

**Reply by glenn-jocher** [reply]
@sangeun-jo thank you for pointing that out. Your modification to `images.extend(sorted(dataset_path.rglob(f"*images/*{ext}")))` is indeed correct and ensures the code works as intended. If you have any further questions or need additional assistance, feel free to ask!
